src/components/List/Cells/ApplicationNameCell.tsx

```diff
@@ -1,11 +1,6 @@
 import React from 'react'
-import { Link } from 'react-router-dom'
 import { CellProps } from '../../../utils/types'
 
-const ApplicationNameCell: React.FC<CellProps> = ({ application }) => (
-  <p>
-    <Link to={`/application/${application.serial}`}>{application.name}</Link>
-  </p>
-)
+const ApplicationNameCell: React.FC<CellProps> = ({ application }) => <p>{application.name}</p>
```

Comment:
Regarding my earlier comment, I think you would put the Link on this.

Reply:
Thanks